esp-wifi/src/ble/btdm.rs

```diff
@@ -408,13 +408,13 @@ unsafe extern "C" fn btdm_sleep_exit_phase3() {
 
 unsafe extern "C" fn coex_schm_status_bit_set(_typ: i32, status: i32) {
     log::debug!("coex_schm_status_bit_set {} {}", _typ, status);
-    #[cfg(coex)]
+    #[cfg(feature = "coex")]
```

Comment:
This is kinda noisy. Perhaps we can keep the old `#[cfg(coex)]` by still outputting the cfg in the build script?

Reply:
I like the idea and also did it for the targets - in theory we could also do it for the other features but for now I think that is fine since we don't use those in `cfg`s that much